specifyweb/stored_queries/format.py

```diff
@@ -335,6 +336,21 @@ def _fieldformat(self, specify_field: Field,
             _case = case(cases)
             return blank_nulls(_case) if self.replace_nulls else _case
         
+        if self.format_picklist:
+            schema_items = Splocalecontaineritem.objects.filter(
+                container__discipline=self.collection.discipline,
+                container__schematype=0,
+                container__name=table.table.lower(),
+                name=specify_field.name.lower(),
+            )
```

Comment:
I think this is fine, but keep in mind that workbench has slightly different logic (it also checks collection picklists). Can you make them behavior same, ideally by wrapping the workbench logic in a single function and then reuse it here to fetch the picklist? Better yet, I'd recommend a function in specifyweb/specify so people later on can reuse this logic.

Reply:
Thanks! That was my initial thought as well but somehow didn't follow through with it. A general function in specifyweb/specify sounds like a great idea